File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 0) return 0;
        int size = nums.size();
        int i = 0;
        for(int j = 0 ; j < size ; j++){
            int c = nums[j];
            while((j+1 < size) && (nums[j+1] == c)){
                j++;
            }
            nums[i] = c ;
            i++;
        }
        return i;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n = len(nums)
        i = 0 
        x = 0
        while x < n  :
            curr = nums[x]
            nums[i] = curr
            while x + 1 < n and nums[x] == nums[x+1] :
                x = x + 1
            x += 1
            i += 1
        return i 
